cmd/internal/shared/devicerepository/config.go

```diff
@@ -14,12 +14,21 @@
 
 package devicerepository
 
-import "go.thethings.network/lorawan-stack/v3/pkg/devicerepository"
+import (
+	"go.thethings.network/lorawan-stack/v3/pkg/devicerepository"
+	"go.thethings.network/lorawan-stack/v3/pkg/devicerepository/store/bleve"
+)
 
 // DefaultDeviceRepositoryConfig is the default configuration for the Device Repository.
 var DefaultDeviceRepositoryConfig = devicerepository.Config{
-	ConfigSource: "url",
-	URL:          "https://raw.githubusercontent.com/TheThingsNetwork/lorawan-devices/master",
+	Source:    "directory",
+	Directory: "data/lorawan-devices",
+
+	Store: devicerepository.StoreConfig{
+		Bleve: bleve.Config{
+			SearchPaths: []string{"device-repository-index", "/srv/ttn-lorawan/device-repository-index"},
```

Comment:
For consistency with `lorawan-devices` in `/data`, can we check in `data` and use `lorawan-devices-index`?